sdk-installer/setup-git-sdk.bat

```diff
@@ -138,7 +138,7 @@
 	mingw-w64-@@ARCH@@-curl mingw-w64-@@ARCH@@-expat ^
 	mingw-w64-@@ARCH@@-openssl mingw-w64-@@ARCH@@-tcl ^
 	mingw-w64-@@ARCH@@-pcre mingw-w64-@@ARCH@@-connect ^
-	git-flow
+	git-flow ssh-pageant
```

Comment:
You probably also want to add it to [this section](https://github.com/landstander668/build-extra/blob/ssh-pageant/make-file-list.sh#L30-L41).